samples/sensor/hx711/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: HX711 24-Bit Analog-to-Digital Converter (ADC) for Weigh Scales
+tests:
+  sample.sensor.hx711:
+    harness: console
+    tags: sensors
+    depends_on: gpio
+    filter: dt_compat_enabled("avia,hx711")
```

Comment:
I don't usually like whitelists, but in this case you need it because there aren't any boards with this compatible. Please replace with `platform_whitelist: nrf52_pca10040`.

That said, didn't this board get renamed recently? @ioannisg ?

Reply:
Yeap, that's now called nrf52dk_nrf52832

Reply:
I do not really understand what **sample.yml** does.

Do I have to change the contents to 
```
sample:
  name: HX711 24-Bit Analog-to-Digital Converter (ADC) for Weigh Scales
tests:
  sample.sensor.hx711:
    platform_whitelist: nrf52dk_nrf52832
```
?

Reply:
`sample.yaml` helps sanitycheck determine which sample applications should be built and tested on which boards.
https://docs.zephyrproject.org/latest/guides/test/sanitycheck.html

Reply:
contents changed to :

```
sample:
  name: HX711 24-Bit Analog-to-Digital Converter (ADC) for Weigh Scales
tests:
  sample.sensor.hx711:
    tags: sensors
    depends_on: gpio
    platform_whitelist: nrf52dk_nrf52832

```

Reply:
Is this change acceptable?
Can this also be resolved now?